### ISSUE TITLE
Create tiles/section for GCP KMS

### ISSUE THREAD

**Issue by mattmoor** [chainguard-dev/terraform-infra-common#65]

**Comment by imjasonh** [comment]
I only see one metric for KMS, "peak qps"
<img width="1333" alt="Screenshot 2024-01-04 at 11 28 09 AM" src="https://github.com/chainguard-dev/terraform-infra-common/assets/210737/e24c79ec-e5fd-4a5e-b4e3-69af022db4d6">


But I guess that'd be useful to see

**Comment by mattmoor** [comment]
Yeah, similar to secrets this was somewhat aspirational.  I'd love metrics around active key count, and key age.
